internal/pkg/cli/svc_init.go

```diff
@@ -604,24 +621,45 @@ func (o *initSvcOpts) askSvcPort() (err error) {
 		return nil
 	}
 
-	port, err := o.prompt.Get(
-		fmt.Sprintf(svcInitSvcPortPrompt, color.Emphasize("port")),
+	if o.wkldType == manifestinfo.RequestDrivenWebServiceType {
+		selectedPorts, err := o.prompt.Get(
+			fmt.Sprintf(svcInitSvcPortPrompt, color.Emphasize("port")),
+			svcInitSvcPortHelpPrompt,
+			validateSvcPort,
+			prompt.WithDefaultInput(defaultPort),
+			prompt.WithFinalMessage("Port:"),
+		)
+		if err != nil {
+			return fmt.Errorf("get port: %w", err)
+		}
+		if len(strings.Split(selectedPorts, ",")) > 1 {
+			return fmt.Errorf("App Runner Service doesn't expose multiple ports")
+		}
+		o.ports = make([]string, 1)
+		o.ports[0] = selectedPorts
+		return nil
+	}
+
+	selectedPorts, err := o.prompt.Get(
+		fmt.Sprintf(svcInitSvcPortPrompt, color.Emphasize("port(s)")),
 		svcInitSvcPortHelpPrompt,
 		validateSvcPort,
 		prompt.WithDefaultInput(defaultPort),
-		prompt.WithFinalMessage("Port:"),
+		prompt.WithFinalMessage("Port(s):"),
 	)
 	if err != nil {
 		return fmt.Errorf("get port: %w", err)
 	}
+	portList := strings.Split(selectedPorts, ",")
```

Comment:
How would users know they need to use comma separator for multiple ports? Do we need a different question?

Reply:
I actually took the refernece of `--import-cert-arns` flag and tried doing the same for consistency purpose. Previously, I thought of doing it as space separated like `3000 3002` so that it is consistent with how we specify ports in dockerfile but then comma separated made more sense in copilot flags world. Happy to get some thoughts here from you.

Reply:
I wonder if we could add a prompt cycle, like so:
```
Which port do you want traffic sent to? 
> 8000
Would you like to add another traffic port?
> Yes
  No
Which port do you want traffic sent to? 
> 50051
Would you like to add another traffic port?
  Yes
> No
```
kind of like we do with Storage Init.

That avoids the need to do complicated parsing and if customers want to avoid it they can use the `--port 8000,50051` or `--port 8000 --port 50051` syntax which would skip the prompt entirely.